slack_bolt/context/complete/complete.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional, Union
+
+from slack_sdk import WebClient
+from slack_sdk.web import SlackResponse
+from slack_bolt.context.complete.internals import can_not_complete, get_kwargs_for_logging, get_name_for_logging
+
+
+class Complete:
+    client: Optional[WebClient]
+    function_execution_id: Optional[str]
+
+    def __init__(
+        self,
+        client: Optional[WebClient],
```

Comment:
Can the `client` argument be None? If not, we can change the type to `WebClient`

Reply:
Good catch, here the client should never be `None` if it is an error should be thrown

Reply:
On `completeSuccess` if outputs is `None` slack interprets it as an error, I added `outputs if outputs else {}` to prevent this error use case